amundsen_application/api/utils/data_utils.py

```diff
@@ -0,0 +1,29 @@
+from typing import Dict
+
+from amundsen_application.api.utils.request_utils import get_query_param
+
+
+def get_table_key(args: Dict) -> str:
+    db = get_query_param(args, 'db')
+    cluster = get_query_param(args, 'cluster')
+    schema = get_query_param(args, 'schema')
+    table = get_query_param(args, 'table')
+    table_key = '{db}://{cluster}.{schema}/{table}'.format(**locals())
+    return table_key
+
+
+def marshall_table_partial(table: Dict) -> Dict:
+    table_name = table.get('table_name')
```

Comment:
table_name = table.get('table_name', ''). The default value is None . Same for all the other string fields.